Remotion/Web/ClientScript/Scripts/ListMenu.ts

```diff
@@ -149,65 +159,59 @@ class ListMenu
 
       if (itemInfo.DiagnosticMetadata)
       {
-        // Do not render empty diagnostic metadata attributes
-        $.each(itemInfo.DiagnosticMetadata, function (key, value)
+        for (const [key, value] of Object.entries (itemInfo.DiagnosticMetadata))
         {
-          if (value === "" || value === null)
+          // Do not render empty diagnostic metadata attributes
```

Comment:
we can drop the comment. this was only needed for the original way to do this. now, it's rather obvious.